### DISCUSSION TITLE
Formatting dutch price

### DISCUSSION THREAD

**Discussion by de-raaf-media** [duncanmcclean/simple-commerce#1140] [Help] [answered]
I am not sure if this is a bug or a configuration issue so I will post it here first as a question. I store my prices in cents in the database. In the front-end my prices are formatted like this:

€4,027.00

In my Laravel configuration I use 'locale' => 'nl',

In the Netherlands. The formatting should be €4.027,00 (dot for thousand separator and a comma for cents).

In the simple-commerce config I use 'currency' => 'EUR',

Is there any way to modify the separator? 

Regards,
Mark

**Comment by duncanmcclean** [comment, marked as answer]
Yes, you can change the separators in your config file:

```php
// config/simple-commerce.php

'sites' => [
  'nl' => [
    'currency' => 'EUR',
    'currency_formatting' => [
      'decimal_separator' => ',',
      'thousand_separator' => '.',
    ],
  ],
],
```

**Reply by de-raaf-media** [reply]
Great! Thanks.
